packages/core/src/sync-store/postgres/migrations.ts

```diff
@@ -492,10 +529,10 @@ const migrations: Record<string, Migration> = {
         checkpoint: bigint;
       }>(
         sql`
-          SELECT blocks.timestamp, blocks."chainId", blocks.number, logs."transactionIndex", logs."logIndex", logs.checkpoint
-          FROM ponder_sync.logs logs
-          JOIN ponder_sync.blocks blocks ON logs."blockHash" = blocks.hash
-          LIMIT 10
+        SELECT blocks.timestamp, blocks."chainId", blocks.number, logs."transactionIndex", logs."logIndex", logs.checkpoint
```

Comment:
FWIW, I think we'd be fine to remove the check here now that we've confirmed this several times.

Reply:
ok yeah makes sense to remove it at this point